src/com/jalasoft/search_files/search/search_engine.py

```diff
@@ -0,0 +1,23 @@
+"""
+class Search_Engine perform the search on a given path of a given file_name applying all the desired filters
+"""
+import os
+
+
+def print_directory(path):
+    return os.listdir(path)
+
+# def search_file(path, file_name):
+#     for root, dirs, files in os.walk(path):
+#         print(root)
+#         # if file_name in files:
+#         #     print("file path directory: %s" % root)
+#         #     print(dirs)
+#         #     print(files)
+#         #     print("exist!!!")
+
+
```

Comment:
This should not be included in the code at least is part of the documentation

Reply:
everything will be refactor on the next branch this was uploaded as an example of use